src/WebappStack.ts

```diff
@@ -189,7 +188,6 @@ export class WebappStack extends Stack {
       environment: {
         SUBMISSION_STORAGE_API_KEY_SECRET_ARN: submissionStorageApiKeySecret.secretArn,
         SUBMISSION_STORAGE_API_BASE_URL: props.configuration.webformsSubmissionsApiBaseUrl,
-        PERMISSION_TABLE_NAME: permissionTable.tableName,
```

Comment:
Deze is niet meer nodig omdat de data in de sessie zit?

Reply:
Die is idd niet nodig. Heb ik helemaal niet gebruikt. Permissies komen uit de session in die lambda. Nog een extra check gedaan. En het werkt allemaal op acceptatie :)